### PR TITLE
[Teams] additions to docs

### PR DESCRIPTION
**Pull request by abracchi-tw**
- Purpose Justification
- Device posture checks on WARP UI
- Fixes to code snippets

### CODE REVIEW

**Review by kennyj42 — APPROVED** [review]

File: products/cloudflare-one/src/content/identity/devices/index.md [hunk 1]
```diff
@@ -12,6 +12,24 @@ When device posture checks are configured, users can only connect to a protected
 
 To enable one or more device posture attributes, navigate to **My Team** > **Devices** > **Device posture** on the [Teams Dashboard](https://dash.teams.cloudflare.com) and click **+Add**.
 
+<Aside>
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/identity/devices/index.md:15]: add `type="note"` to `<Aside>`

File: products/cloudflare-one/src/content/policies/zero-trust/require-purpose-justification.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+order: 1
+pcx-content-type: how-to
+title: Require Purpose Justification
+---
+
+# Require Purpose Justification After Login
+
+Cloudflare Access allows security and IT teams to present users with a purpose justification screen directly after they log into an Access application. This allows organizations to audit not only for *who* is accessing their resources, but also for *why* they are requesting access.
+
+The purpose justification screen will show for any new sessions of an application. For example, if an Access application has a session time of 8 hours, a user will see the purpose justification screen once every 8 hours.
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/policies/zero-trust/require-purpose-justification.md]: should it be `eight hours` instead?

File: products/cloudflare-one/src/content/policies/zero-trust/require-purpose-justification.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+Configuring a purpose justification screen is done as part of configuring an Access policy.
+
+1. On the Teams Dashboard, navigate to **Access** > **Applications**.
+1. Select an application and click **Edit**.
+1. Select the policy you want to configure with purpose justification
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/policies/zero-trust/require-purpose-justification.md]: Add `.` at the end of the sentence

File: products/cloudflare-one/src/content/policies/zero-trust/require-purpose-justification.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+1. Open **Optional Configurations**.
+1. Enable purpose justification
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/policies/zero-trust/require-purpose-justification.md]: Add `.` at the end of the sentence

File: products/cloudflare-one/src/content/policies/zero-trust/require-purpose-justification.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+1. (Optional) set a custom purpose justification message. This will appear on the purpose justification screen and will be visible to the user.
+
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/policies/zero-trust/require-purpose-justification.md]: Delete extra space

**Review by marciocloudflare — APPROVED** [review]

**Pull request merged**